Main_menu.py

```diff
@@ -0,0 +1,59 @@
+import pygame
+import sys
+import pygame.freetype
+
+red = (255, 0, 0)
+green = (0, 255, 0)
+blue = (0, 0, 255)
+yellow = (255, 255, 0)
+black = (0, 0, 0)
+white = (255, 255, 255)
+
+
+class MainMenu(object):
+    def __init__(self, game):
+        self.game = game
+        self.size = self.game.screen.get_size()
+        self.menu = pygame.freetype.SysFont('Arial', 50)
```

Comment:
rozmiary do osobnych zmiennych

Reply:
Konwencja w pythonie,  _ protected, __ private (https://www.tutorialsteacher.com/python/private-and-protected-access-modifiers-in-python). Zmienne które nie będą używane poza klasą powinny być private i mieć __ na początku nazwy

Reply:
menu jest mało opisowe, szczególnie jak dalej przypisujesz do niego jakiś font. Nazwa do zmiany